webapp/src/components/App/Singlestudy/explore/Modelization/Map/MapConfig/Districts/index.tsx

```diff
@@ -85,6 +85,9 @@ function Districts() {
           output: districtsById[districtId].output,
           comments: districtsById[districtId].comments,
           areas: areasByDistrict[districtId],
+          // remove-all is hard coded here
+          // The front always send updates with apply filter set to remove-all
+          applyFilter: "remove-all",
```

Comment:
Note that this changes the behaviour when you update an area:
for example, for the district "all areas", if you remove one area from the district, then add new areas in the study, they will not be part of the district anymore.

We'll not clarify this in that PR though